examples/azure-wordpress/params.ini

```diff
@@ -1,2 +1,2 @@
-mysql_password=ilovecnab
-server_name=ilovecnab
\ No newline at end of file
+mysql_password=iloveporter-127c6!J9$
```

Comment:
lol strong stance there 😉 Take that CNAB

Reply:
😂 Also added those extry chars b/c Azure was like 'password not complex enuff'